LayoutTests/imported/w3c/web-platform-tests/css/css-properties-values-api/registered-property-computation.html

```diff
@@ -154,15 +155,15 @@
 test_computed_value('<color>', '#badbee33', 'rgba(186, 219, 238, 0.2)');
 test_computed_value('<color>', 'tomato', 'rgb(255, 99, 71)');
 test_computed_value('<color>', 'plum', 'rgb(221, 160, 221)');
-test_computed_value('<color>', 'currentcolor', 'currentcolor');
+test_computed_value('<color>', 'currentcolor', 'rgb(0, 0, 255)');
```

Comment:
Do we have tests for the specified values of custom properties somewhere? I think we'd still want to test that this returns currentColor. Ditto for the testcases below.